src/Operators/Operators.jl

```diff
@@ -4,12 +4,16 @@ export
     Δx, Δy, ΔzF, ΔzC,
     Δxᶜᶜᵃ, Δxᶠᶜᵃ, Δxᶠᶠᵃ, Δxᶜᶠᵃ,
     Δyᶜᶜᵃ, Δyᶠᶜᵃ, Δyᶠᶠᵃ, Δyᶜᶠᵃ,
-    Axᵃᵃᶜ, Axᵃᵃᶠ, Axᶠᶜᶜ,
-    Ayᵃᵃᶜ, Ayᵃᵃᶠ, Ayᶜᶠᶜ,
+    Axᵃᵃᶜ, Axᵃᵃᶠ, Axᶜᶜᶜ, Axᶠᶜᶜ, Axᶠᶠᶜ, Axᶠᶜᶠ,
+    Ayᵃᵃᶜ, Ayᵃᵃᶠ, Ayᶜᶜᶜ, Ayᶜᶠᶜ, Ayᶠᶠᶜ, Ayᶜᶠᶠ,
     Azᵃᵃᵃ, Azᶠᶠᵃ, Azᶜᶜᵃ, Azᶠᶜᵃ, Azᶜᶠᵃ,
-    Vᵃᵃᶜ, Vᵃᵃᶠ, Vᶜᶜᶜ,
+    Vᵃᵃᶜ, Vᵃᵃᶠ, Vᶜᶜᶜ, Vᶠᶜᶜ, Vᶜᶠᶜ, Vᶜᶜᶠ,
     δxᶜᵃᵃ, δxᶠᵃᵃ, δyᵃᶜᵃ, δyᵃᶠᵃ, δzᵃᵃᶜ, δzᵃᵃᶠ,
-    Ax_ψᵃᵃᶠ, Ax_ψᵃᵃᶜ, Ay_ψᵃᵃᶠ, Ay_ψᵃᵃᶜ, Az_ψᵃᵃᵃ, Ax_uᶠᶜᶜ, Ay_vᶜᶠᶜ, Az_wᶜᶜᵃ,
+    Ax_ψᵃᵃᶠ, Ax_ψᵃᵃᶜ, Ay_ψᵃᵃᶠ, Ay_ψᵃᵃᶜ, Az_ψᵃᵃᵃ,
+    Ax_uᶠᶜᶜ, Ay_vᶜᶠᶜ, Az_wᶜᶜᵃ,
+    Ax_ζᶠᶠᶜ, Ay_ζᶠᶠᶜ, Ax_ηᶠᶜᶠ, Az_ηᶠᶜᵃ, Ay_ξᶜᶠᶠ, Az_ξᶜᶠᵃ,
+    Az_wᶜᶜᵃ, Az_ηᶠᶜᵃ, Az_ξᶜᶠᵃ,
+    Ax_cᶜᶜᶜ, Ay_cᶜᶜᶜ, Az_cᶜᶜᵃ,

```

Comment:
`u`, `v`, `w`, and `c` should be pretty clear but just wondering if there's a pattern to using `ψ` vs. `ζ` vs. `η` vs. `ξ`?

Reply:
I'm defining the components of vorticity as `(ξ, η, ζ)` (eg the Greek x, y, z for the x, y, z components). I was thinking that adopting specific notation might help us remember and talk about these locations, and also could help avoid bugs? What do you think?

Reply:
Ah that makes sense! I think it's a good idea but might be good to add a comment somewhere explaining the choice of function names? What about `ψ`?